scripts/src/test.sh

```diff
@@ -148,6 +149,11 @@ run_fuzzer() {
   cargo fuzz run --release "$FEATURES" --sanitizer=none "$TARGET"
 }
 
+# TODO this is likely to be merged with `pallet_test` or `work_space` in #1802
```

Comment:
```suggestion
# TODO this is likely to be merged with `pallet_test` or `workspace` in #1802
```

Reply:
even should be `workspace_test`. will do

Reply:
done.